.github/workflows/release.yml

```diff
@@ -0,0 +1,149 @@
+name: Release
```

Comment:
I don't think we want or need this pipeline. I'd rather still manually craft a release in GitHub instead of making it automatic on tag push